Packs/ThreatIntelligenceManagement/Scripts/IncidentsPerFeed/IncidentsPerFeed.yml

```diff
@@ -0,0 +1,37 @@
+args:
+  - default: false
+    description: Additional filters for the 'getIncidents' command
+    isArray: false
+    name: query
+    required: false
+    secret: false
+  - default: false
+    description: Name to display in the widget as the distinct incidents name
```

Comment:
Similar to what I wrote in the code, I think both the argument name here, and its description can be improved.